armi/tests/armiRun.yaml

```diff
@@ -19,11 +19,6 @@ settings:
       numInternalRings: 1
       numExternalRings: 1
     XA:
-      geometry: 0D
-      externalDriver: true
-      useHomogenizedBlockComposition: false
-      numInternalRings: 1
-      numExternalRings: 1
```

Comment:
This is removed because we should only be defining `fileLocation` OR `geometry`, not both